page_change.go

```diff
@@ -0,0 +1,34 @@
+package eventstream
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// RevisionCreate event scheme struct
+type PageChange struct {
+	baseSchema
+	Data struct {
+		baseData
+		Dt            time.Time `json:"dt"`
+		ChangelogKind string    `json:"changelog_kind"`
+		Page          struct {
+			PageID    int    `json:"page_id"`
+			PageTitle string `json:"page_title"`
+		} `json:"page"`
+		Revision struct {
+			RevID int       `json:"rev_id"`
+			RevDt time.Time `json:"rev_dt"`
+		} `json:"revision"`
+		WikiID string `json:"wiki_id"`
+	}
+}
```

Comment:
There is a lot of schema changes between the older streams (such as [revision-create)](https://schema.wikimedia.org/repositories//primary/jsonschema/mediawiki/revision/create/latest.yaml) vrs the new [page-change](https://schema.wikimedia.org/repositories//primary/jsonschema/mediawiki/page/change/latest.yaml) stream.

In order to accommodate both, I suggest the following changes:

1. Include `page_change_kind` as a PageChange.Data field. 

2. As you can see in both the schemas, revision fields are restructured quite a bit.
In the old schema, revision fields (such as rev_id) is in the root. In the new schema, the revision fields are structured under a revision struct.
To work with both, take out `RevID` from the `baseData`. Add `RevID` to all the old structs (such as RevisionCreate).Data as a field.
For the new struct PageChane.Data.Revision add all the following fields:
     - rev_id
     - rev_dt
     - comment
     - content_slots.content_format
     - content_slots.content_model
     - is_comment_visible
     - is_content_visible
     - is_editor_visible
     - is_minor_edit
     - rev_parent_id
     - rev_sha1
     - rev_size
     
 3. Page fields are restructured as well. Remove the following from `baseData`:
      
```
        PageID         int       `json:"page_id"`
	PageTitle      string    `json:"page_title"`
	PageNamespace  int       `json:"page_namespace"`
	PageIsRedirect bool      `json:"page_is_redirect"`
```

	
Add these to all the older structs (such as RevisionCreate).Data as a field there.
	
For the new struct (PageChange), also include the missing namespace and redirect.

4. `Database` is renamed in the new schema. Remove the following from `baseData`:
       
       `Database       string    `json:"database"``
       
  Add it to all the older structs (such as RevisionCreate).Data as a field there.
       
  For the new struct (PageChange), add the following under .Data
       
      ` Database       string    `json:"wiki_id"``